picard/ui/mainwindow/__init__.py

```diff
@@ -780,7 +788,7 @@ def add_toolbar_action(action_id):
         self.show_toolbar()
 
     def _create_player_toolbar(self):
-        """"Create a toolbar with internal player control elements"""
+        """ "Create a toolbar with internal player control elements"""
```

Comment:
Fixed in https://github.com/metabrainz/picard/pull/2697/commits/2f533f555ca3c269ff53ee9ba77066e3191fe38b